src/lib/functions/server.js

```diff
@@ -107,10 +107,12 @@ const createHandler = function ({ functionsRegistry }) {
 
       handleBackgroundFunctionResult(functionName, error)
     } else if (await func.isScheduled()) {
-      if (event.headers['user-agent'] !== 'netlify-cli') {
+      if (event.headers['user-agent'] !== 'Netlify Clockwork') {
```

Comment:
[sand] Since this value is used not only to mimic the production behaviour but also to decide whether a function invocation is from the scheduler or not, should we move it to a constant to avoid a scenario where someone changes it in one place but not the other?

Reply:
https://github.com/netlify/cli/pull/3689/commits/17798030fa0250238258cfab8e69438c07acdb42